C/c4ExpiryEnumerator.h

```diff
@@ -19,24 +19,24 @@ extern "C" {
     /** Opaque handle to an enumerator that iterates through expired documents. */
     typedef struct C4ExpiryEnumerator C4ExpiryEnumerator;
     
-    /** Creates an enumerator for iterating over expired documents
-     Caller is responsible for freeing the enumerator when finished with it.
-     @param database  The database.
-     @param outError  Error will be stored here on failure.
-     @return  A new enumerator, or NULL on failure. */
+	/** Creates an enumerator for iterating over expired documents
```

Comment:
Indentation is wrong in all these lines — make sure your editor uses spaces not tabs for indenting.